data/mods/MindOverMatter/powers/telepathy_eoc.json

```diff
@@ -1,4 +1,118 @@
 [
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_TELEPATH_MORALE_INITIATE",
+    "condition": "u_is_avatar",
+    "effect": [ { "run_eocs": "EOC_TELEPATH_MORALE_INITIATE_SELF" } ],
+    "false_effect": [
+      {
+        "run_eocs": [
+          {
+            "id": "EOC_TELEPATH_MORALE_INITIATE_NPC_CHECK",
+            "condition": "u_is_npc",
+            "effect": [
+              { "math": [ "u_telepathy_intelligence", "=", "( ( n_val('intelligence') + 10) / 20 )" ] },
+              { "math": [ "u_telepathy_power_level", "=", "n_spell_level('telepathic_morale')" ] },
+              { "math": [ "u_nether_attunement_telepathy_scaling", "=", "n_nether_attunement_power_scaling" ] },
+              {
+                "math": [
+                  "u_mood_stabilized_power_level",
+                  "=",
+                  "u_telepathy_power_level * u_telepathy_intelligence * u_nether_attunement_telepathy_scaling"
+                ]
+              },
+              { "run_eocs": "EOC_TELEPATH_MORALE_MODIFY_NPC_VALUES" }
+            ],
+            "false_effect": [
+              { "math": [ "u_telepathy_intelligence", "=", "( ( n_val('intelligence') + 10) / 20 )" ] },
+              { "math": [ "u_telepathy_power_level", "=", "n_spell_level('telepathic_morale')" ] },
+              { "math": [ "u_nether_attunement_telepathy_scaling", "=", "n_nether_attunement_power_scaling" ] },
+              {
+                "math": [
+                  "u_mood_stabilized_power_level",
+                  "=",
+                  "u_telepathy_power_level * u_telepathy_intelligence * u_nether_attunement_telepathy_scaling"
+                ]
+              },
+              { "run_eocs": "EOC_TELEPATH_MORALE_MODIFY_MONSTER_VALUES" }
+            ]
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_TELEPATH_MORALE_MODIFY_NPC_VALUES",
+    "condition": { "not": { "u_has_effect": "effect_telepath_mood_stabilization_attitude_modified" } },
+    "effect": [
+      {
+        "run_eocs": [
+          {
+            "id": "EOC_TELEPATH_MORALE_MODIFY_NPC_VALUES_2",
+            "condition": { "not": { "u_has_effect": "effect_telepathic_psi_armor" } },
+            "effect": [
+              { "math": [ "u_pre_power_fear", "=", "u_val('npc_fear')" ] },
```

Comment:
you do not need to store npc values before the effect
```json
{ "math": [ "u_val('npc_fear')", "=", "max( -10, (u_val('npc_fear') - (u_mood_stabilized_power_level / 2)))" ] },
```
this would work just fine